Cannot find module 'opentracing'

Opentracing support (#6347) was recently merged and released in the latest beta of gatsby, but after updating my dependencies with `yarn upgrade` I'm experiencing the following error:

```
$ gatsby develop
module.js:540
    throw err;
    ^

Error: Cannot find module 'opentracing'
    at Function.Module._resolveFilename (module.js:538:15)
    at Function.Module._load (module.js:468:25)
    at Module.require (module.js:587:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/home/maxime/react/lebonforfait/web/node_modules/gatsby-cli/lib/reporter/index.js:11:16)
    at Module._compile (module.js:643:30)
    at Object.Module._extensions..js (module.js:654:10)
    at Module.load (module.js:556:32)
    at tryModuleLoad (module.js:499:12)
    at Function.Module._load (module.js:491:3
```

It seems that there is a missing dependency somewhere.

--- a/packages/gatsby-cli/package.json
+++ b/packages/gatsby-cli/package.json
@@ -22,6 +22,7 @@
     "fs-extra": "^4.0.1",
     "hosted-git-info": "^2.6.0",
     "lodash": "^4.17.4",
+    "opentracing": "^0.14.3",
     "pretty-error": "^2.1.1",
     "resolve-cwd": "^2.0.0",
     "source-map": "^0.5.7",

